marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matching.cy.tsx

```diff
@@ -370,8 +370,10 @@ describe("Matching", () => {
     cy.wait(1000);
     cy.findByText("At least Two URIs are required.").should("be.visible");
     matchingStepDetail.getUriOnlyInputField().clear().type("/test/Uri1");
-    matchingStepDetail.getAddUriOnlyIcon().click();
-    cy.findByText("This URI has already been added.").should("be.visible");
+    matchingStepDetail.getAddUriOnlyIcon().click().then(() => {
+      cy.get("[aria-label='UriOnlyInput']").should("have.attr", "placeholder", "Enter URI or Paste URIs");
```

Comment:
I believe that the cy get functions should be in the pages section. There already is a getUriOnlyInputField method in  matchingStepDetail you can invoke.

Reply:
oops, fixed :)